### DISCUSSION TITLE
Enable/Disable "Polling for updates"

### DISCUSSION THREAD

**Discussion by alexdelprete** [frenck/spook#42] [Ideas] [answered]
Frenck, only the anarchic Spook could save the world by spooking a service for this:

![image](https://user-images.githubusercontent.com/7027842/223583084-08af7fb4-2d96-4036-b641-13b84e49dc61.png)

**Comment by frenck** [comment]
Nice suggestion 👍

**Comment by frenck** [comment, marked as answer]
Added in https://github.com/frenck/spook/pull/46

**Reply by alexdelprete** [reply]
Thank you Frenck. I hope some of these spooky functions will be part of core one day. :)

**Reply by frenck** [reply]
Probably not :)

**Reply by alexdelprete** [reply]
> Probably not :)

too dangerous? :)

**Reply by frenck** [reply]
No, it is managing configuration, for which we don't allow services in Core.

**Reply by frenck** [reply]
released in v0.11.1112

**Reply by alexdelprete** [reply]
> released in v0.11.1112

Thank you Frenck.
